demo/build.gradle

```diff
@@ -4,16 +4,16 @@ apply plugin: 'kotlin-kapt'
 apply plugin: 'kotlin-android-extensions'
 
 android {
-  compileSdkVersion 27
-
+  compileSdkVersion 28
   defaultConfig {
     applicationId "se.springworks.mvvmcomponents.demo"
-    minSdkVersion 18
-    targetSdkVersion 27
+    minSdkVersion 19
```

Comment:
Should we change to 19 here?
But we are using 19 in the app so no problem for us right?

Reply:
yes we support 19 in the app